docs/guides/expectations/creating_custom_expectations/overview.md

```diff
@@ -1,18 +1,76 @@
+
+| Guide: "How to create a custom..." |  Template |
+|-----------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [Column Map Expectation](how_to_create_custom_column_map_expectations)             | [column_map_expectation_template](https://github.com/great-expectations/great_expectations/blob/develop/examples/expectations/column_map_expectation_template.py)       |
+| [Column Aggregate Expectation](how_to_create_custom_column_aggregate_expectations) | [column_aggregate_expectation_template](https://github.com/great-expectations/great_expectations/blob/develop/examples/expectations/column_map_expectation_template.py) |
+
+Note: not all classes of Expectation currently have guides and templates. If you'd like to develop a different kind of Expectation, please reach out on [Slack](https://greatexpectations.io/slack).
+
+Beyond the first four steps, additional features are generally similar across all Expectation classes. Accordingly, each of the remaining steps has its own how-to guide in the `Adding features to Great Expectations` section of the table of contents.
+
+| Step | Guide |
+|------|-------|
+| Has basic input validation and type checking                                                     | [How to add configuration validation for an Expectation](../features_custom_expectations/how_to_add_input_validation_for_an_expectation) |
+| Has all four statement `Renderers`: question, descriptive, prescriptive, diagnostic              | [How to add text renderers for custom Expectations](../features_custom_expectations/how_to_add_text_renderers_for_an_expectation) |
+| Has default `Parameter Builders` and Domain hooks to support Profiling                           | |
+| Has core logic exists and passes tests for all applicable `Execution Engines` and SQL dialects   | [How to add SQLAlchemy support for custom Metrics](../features_custom_expectations/how_to_add_sqlalchemy_support_for_an_expectation)<br/> [How to add Spark support for custom Metrics](../features_custom_expectations/how_to_add_spark_support_for_an_expectation)|
+| Passes all linting checks                                                                        | |
+| Has all applicable Renderers, with fully typed and styled output                                 | |
+| Has a full suite of tests, as determined by project code standards                               | |
+| Has passed a manual review by a code owner for code standards and style guides                   | |
+
+## Publishing your Expectation as an open source contribution
+
+(Add this later; possibly in a different location in the TOC)
```

Comment:
Could you link to: contributing/contributing for now?

Reply:
Sure, although these docs won't go live until after another merge from `hackathon-docs` to `develop`.

Reply:
```suggestion
You can find more detailed instructions in the [Contributing](/docs/contributing/contributing) section of the docs.
```